src/components/panels/OffersPanel/OffersPanel.tsx

```diff
@@ -0,0 +1,137 @@
+import { ReactElement, useState } from "react";
+import useModelStatus from "hooks/useModelStatus";
+
+import { generatePanelTableRows } from "../shared";
+
+import "../_panels.scss";
+
+type Props = {
+  entity: string;
+};
+
+export default function OffersPanel({
+  entity: application,
+}: Props): ReactElement {
+  const modelStatusData = useModelStatus();
+  const offers = modelStatusData?.offers[application];
+
+  const [accordionPanelId, setAccordionPanelId] = useState("");
+
+  const handleAccordionToggle = (id: string) => {
+    setAccordionPanelId(id === accordionPanelId ? "" : id);
+  };
+
+  interface tableData {
+    th: string;
+    td: string;
+  }
+
+  const offerInfo: tableData[] = [
+    { th: "Offer name", td: offers["offer-name"] },
+    { th: "Application", td: application },
+    { th: "Charm", td: "-" },
+    { th: "Store", td: "-" },
+    { th: "URL", td: "-" },
+  ];
+
+  const tab1: tableData[] = [
+    { th: "Endpoint", td: "-" },
+    { th: "Interface", td: "-" },
+    { th: "Role", td: "-" },
+  ];
+
+  const tab2: tableData[] = [
+    { th: "UUID", td: "-" },
+    { th: "User", td: "-" },
+    { th: "Relation-id", td: "-" },
+    { th: "Status", td: "-" },
+    { th: "Ingress-subnets", td: "-" },
+  ];
+
+  const tab3: tableData[] = [
+    { th: "User", td: "-" },
+    { th: "display-name", td: "-" },
+    { th: "access", td: "-" },
+  ];
+
+  return (
+    <div className="panel">
+      <span className="p-muted-heading">mysql:db</span>
```

Comment:
There are a number of hardcoded values here, we'll have to be sure to go back and update them all before this goes live.